### ISSUE TITLE
[Bug]: Legacy Code Signing not working when using useCompileFolder and doNotPublishApps

### ISSUE THREAD

**Issue by jonaswre** [microsoft/AL-Go#1340]
### AL-Go version

V6.0

### Describe the issue

Is there a reason why apps are not signed when you use useCompilerFolder and the legacy Signing? 

https://github.com/microsoft/navcontainerhelper/blob/922b3560177961affbf4b34852c563e54b00ff49/AppHandling/Run-AlPipeline.ps1#L2218

I've checked the the new code signing it works as expected. 

### Expected behavior

Sign the apps even when they are only compiled. 

### Steps to reproduce

Use legacy code signing with useCompilerFolder

**Comment by jonaswre** [comment]
So the solution for our partner is to purchase a new certificate even though their current certificate is still valid for almost a year or pay a lot for github runner minutes?

**Comment by freddydk** [comment]
No, the solution would be to upload your existing certificate to an Azure KeyVault if you want to use the compilerFolder feature.

**Comment by jonaswre** [comment]
> No, the solution would be to upload your existing certificate to an Azure KeyVault and use the new code signing functionality if you want to use the compilerFolder feature.
> 

Ahh cool, didn't know you could do that. 

Thanks for the clarification.
